### PR TITLE
Add test cases for all Header Component and empty state components

### PR DESCRIPTION
**Pull request by drcoolsanjeev**
### Description

Add test case for following components:-
- AddressDetailHeader
- AddressSapceHeader
- ConnectionDetailHeader
- EmptyAddresses
- EmptyConnections

### CODE REVIEW

File: console/console-init/ui/src/Tests/AddressSapceHeader.test.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    };
+
+    const { getByText } = render(<AddressSpaceHeader {...props} />);
+
+    const nameNode = getByText(props.name);
+    const namespaceNode = getByText(props.namespace);
+    const typeNode= getByText(props.type);
+    const createdOnNode = getByText(props.createdOn);
+
+    expect(nameNode).toBeDefined();
```
Comment (riccardo-forina) [inline console/console-init/ui/src/Tests/AddressSapceHeader.test.tsx]: no need to check for this explicitly, all the `get*` methods will throw an error if the searched content is not found.

File: console/console-init/ui/src/Components/AddressSpace/AddressSpaceHeader.tsx [hunk 2]
```diff
@@ -22,15 +22,15 @@ const styles = StyleSheet.create({
...
+export interface IAddressSpace {
   name: string;
   namespace: string;
   createdOn: string;
   type: string;
   onDownload: (name: string) => void;
   onDelete: (name: string) => void;
 }
-export const AddressSpaceHeader: React.FunctionComponent<AddressSpace> = ({
+export const AddressSpaceHeader: React.FunctionComponent<IAddressSpace> = ({
```
Comment (riccardo-forina) [inline console/console-init/ui/src/Components/AddressSpace/AddressSpaceHeader.tsx:33]: `IAddressSpaceProps` please, for consistency

**Review by riccardo-forina — CHANGES_REQUESTED** [review]

File: console/console-init/ui/src/Tests/AddressSapceHeader.test.tsx (hunk 1 above)
Comment (riccardo-forina) [reply console/console-init/ui/src/Tests/AddressSapceHeader.test.tsx]: The same comment applies to the rest of the changes of course.

Comment (drcoolsanjeev) [reply console/console-init/ui/src/Tests/AddressSapceHeader.test.tsx]: Okay, Then what exactly do we need to test in these scenarios ?

**Comment by drcoolsanjeev** [comment]
@riccardo-forina , made the changes as requested. Please let me know if any other change is required.

**Review by riccardo-forina — APPROVED** [review]
